lib/rails_data_migrations/version.rb

```diff
@@ -1,3 +1,3 @@
 module RailsDataMigrations
-  VERSION = '1.2.0'.freeze
+  VERSION = '1.2.0.1'.freeze
```

Comment:
Is this not worthy of '1.2.1' or even a '1.3'?

Reply:
I didn't want to go to 1.3 because that is the current version of the repo we're forked from. So I had opted for this as a safe version that wouldn't conflict. But it doesn't look like 1.2.1 was ever used so i'll switch to that